lib/Mojo/IOLoop/Subprocess.pm

```diff
@@ -124,7 +124,7 @@ can emit the following new ones.
   });
 
 Emitted in the parent process when the subprocess calls the
-L<progress|/"progress1"> method.
+L<progress|/"progress"> method.
```

Comment:
This doesn't really work, as there are two 'progress' sections.  The progress1 link is meant to link to the second section, documenting the method.  There isn't really a good way to "properly" link to a section like that.  It's up to however the Pod renderer decides to act.

Reply:
> This doesn't really work, as there are two 'progress' sections. The progress1 link is meant to link to the second section, documenting the method. There isn't really a good way to "properly" link to a section like that. It's up to however the Pod renderer decides to act.

I have removed the change.